src/components/Navbar/index.jsx

```diff
@@ -55,6 +56,7 @@ export default class Navbar extends React.Component {
 						<div id="hamburger-menu">
 							<ul className="nav-items">
 								<NavLink to="/about"><li>About</li></NavLink>
+								<NavLink to="/media"><li>media</li></NavLink>
```

Comment:
Wait actually good catch here, I forgot about this and I don't think anyone else did this.